packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.8.7+2
 
 * Updates minimum Flutter version to 3.3.
+* Fixed a broken link in the example related to `image_picker_for_web`.
```

Comment:
Nit: Fixes

See the style guide for CHANGELOGs listed in the PR checklist.

Reply:
my bad, I went through it yesterday but I misread it. Sorry